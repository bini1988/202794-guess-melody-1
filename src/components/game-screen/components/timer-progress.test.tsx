import React from "react";
import renderer from "react-test-renderer";
import serializer from "jest-emotion";
import TimerProgress from "./timer-progress";

expect.addSnapshotSerializer(serializer);

it(`TimerProgress correctly renders default markup`, () => {
  const tree = renderer
    .create(
        <TimerProgress
          progress={50}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
