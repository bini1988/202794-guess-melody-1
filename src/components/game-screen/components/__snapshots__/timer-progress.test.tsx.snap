// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimerProgress correctly renders default markup 1`] = `
.emotion-0 {
  position: absolute;
  left: 8px;
  top: 56px;
  width: 665px;
  height: 665px;
  z-index: 0;
}

<svg
  className="emotion-0"
  viewBox="0 0 780 780"
>
  <circle
    cx="390"
    cy="390"
    r={370}
    style={
      Object {
        "fill": "transparent",
        "filter": "url(#blur)",
        "stroke": "#ff9749",
        "strokeDasharray": 2324.7785636564467,
        "strokeDashoffset": 1162.3892818282234,
        "strokeLinecap": "round",
        "strokeWidth": "15px",
        "transform": "rotate(-90deg)",
        "transformOrigin": "center",
      }
    }
  />
</svg>
`;
